### PR TITLE
PICARD-305: Add files to 'Pending Files' status on Lookup action

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

File: picard/file.py [hunk 1]
```diff
@@ -511,6 +511,7 @@ def _lookup_finished(self, lookuptype, document, http, error):
         if self.state == File.REMOVED:
             return
 
+        self.clear_pending()
```
Comment (mineo) [inline picard/file.py]: Why is this necessary? AFAICT all paths through this function call `clear_pending` already.

**Comment by samj1912** [comment]
Updated :)

**Comment by Sophist-UK** [comment]
Resolves: [PICARD-305](https://tickets.metabrainz.org/browse/PICARD-305)
